src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import java.util.Iterator;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Association class for tracking the addition/deletion of Tag onto Person.
+ */
+public class Tagging {
+    
+    public final boolean isAddingTag;
```

Comment:
Not recommended to use a boolean here. It is not obvious the other case is deletion, and you cannot easily extend it.